queries/pokemon.graphql

```diff
@@ -0,0 +1,23 @@
+query Pokemon($id: String!) {
+  pokemon(id: $id) {
+    id
+    number
+    name
+    weight {
+      minimum
+      maximum
+    }
+    height {
+      minimum
+      maximum
+    }
+    classification
+    types
+    resistant
+    weaknesses
+    fleeRate
+    maxCP
+    maxHP
+    image
+  }
+}
```

Comment:
Moved the `FeaturedPokemon` query, to this generic Pokemon query by $id. We can reuse this for the `pokedex/[id]` page.